governance.md

```diff
@@ -57,6 +57,22 @@ Qualification guidelines
 
 If you’d like to nominate yourself as an Approver, please create an [Issue in the community repository](https://github.com/spinnaker/community/issues). 
 
+### SIG Leads
```

Comment:
How is this intended to relate to the "SIGS" section later in the doc? https://www.spinnaker.io/community/governance/#sigs

It states there that "SIGs and their leads are ratified and greenlit by the steering committee", which isn't in conflict with the qualifications described here, but adds an additional requirement (ratification by the SC) that isn't mentioned here.

The [SIGS page](https://www.spinnaker.io/community/governance/sigs/) also mentions that "Each SIG has one or more “leads”, who own the SIG’s agenda", which, while again not in conflict, is a different description of the duties of a SIG lead as described here.

Thoughts on one or either of:

* updating the description of SIGS later in this doc to refer to this new role
* updating the SIGs page to link to the new role description

?

To also take a step back: is there a particular issue/set of issues we're looking to address by formalizing the role in this way?

Reply:
I actually think there is a conflict between line 74 below, "Self-nominated, elected by SIG attendees", and the line in the "SIGS" section of governance document, "SIGs and their leads are ratified and greenlit by the steering committee".

I like both of Andrew's suggestions and I am also interested in the answer to his question.

Reply:
SIG Lead nomination: 

I'm happy to fall in-line with the rest of the docs so that Leads are always ratified by the SC. As we get more SIGs and the community grows, I fear gating all SIG Lead changes by the SC could stall momentum. 

By making SIG Lead nomination and election community-driven, a SIG could be a little more autonomous than they are today. We'd still likely want more language around how many Leads must be maintained by a SIG and around diversity of represented invested organizations (such that a company can't steamroll leadership of a SIG).

Sidenote: If ratification is required for SIG Lead changes, we'll also need to explore term limits and replacement processes, which it looks like we're missing today. 

> To also take a step back: is there a particular issue/set of issues we're looking to address by formalizing the role in this way?

This is in response to direct feedback by the SIG Leads to the TOC in that they don't have a clear understanding of what they are supposed to be doing for the community or their SIG(s), as well as ghosting / too-light-touch SIG Leads. SIG Leads that have been around since the beginning understand, but only due to tribal knowledge.

There has been some impression that SIG Leads would only be responsible for setting up and running the SIG meetings, when in fact there's a lot more project and product management tasks that they need to perform to have a SIG be effective. Prior to this, a lot of day-to-day responsibility was implicitly falling on the TOC that doesn't belong.

Reply:
Taking a step back, perhaps SC ratification of SIG Leads is best for the short-/medium-term while SIG attendance is low. 🤔

Reply:
I also think it's ok to leave ratification with the SC for now. Maybe we can add a standing SC meeting agenda item to ensure we work through any vote-ready issues in each meeting.

Reply:
> This is in response to direct feedback by the SIG Leads to the TOC in that they don't have a clear understanding of what they are supposed to be doing for the community or their SIG(s), as well as ghosting / too-light-touch SIG Leads

Ack, thanks for the additional details!